upi/openstack/inventory.yaml

```diff
@@ -70,3 +70,7 @@ all:
       # Be aware that the 10 and 11 of the machineNetwork will
       # be taken by neutron dhcp by default, and wont be available.
       os_ingressVIP: "{{ os_subnet_range | ansible.utils.next_nth_usable(7) }}"
+
+      # Dual stack installation. When set to True, additional security group
+      # rules will be created.
+      os_dualstack: False
```

Comment:
Can we discover if the cluster is dual-stack from the install-config.yaml or from the generated manifests?

Reply:
maybe we could do that once/if we add v6 support for `os_subnet_range` ?

Reply:
IMHO the only way to distinguish dualstack installation from single stack ipv4 is to look at install-config.yaml and find out if there are at least two networks for machineNetwork (also clusterNetwork and serviceNetwork) one for ipv4 and one for ipv6. Let me check if I could get something out of it.

Reply:
OTOH, there is an inventory, which contain user provided information. So, perhaps instead of having single `os_subnet_range`, it might be a list of subnets:
```yaml
  os_subnets_range:
    - 10.0.0.0/16
    - "fd2e:6f44:5dd8:c956::/64"
```
and starting from here we could distinguish dualstack installation?